Examples/PcapSplitter/Splitters.h

```diff
@@ -1,6 +1,10 @@
 #pragma once
 
 #include <LRUList.h>
+
+#if !defined(WIN32) && !defined(WINx64) && !defined(PCAPPP_MINGW_ENV)
+#include <netinet/in.h>
+#endif
```

Comment:
why are those lines needed here?

Reply:
The following compilation errors occur under Linux:
```
==> Building target: PcapSplitter
Building file: main.cpp
In file included from Splitters.h:8,
                 from SimpleSplitters.h:3,
                 from main.cpp:56:
/home/ssuslov/src/PcapPlusPlus/Dist/header/IpUtils.h:63:2: error: ‘in_addr’ does not name a type; did you mean ‘s6_addr’?
  in_addr* sockaddr2in_addr(struct sockaddr *sa);
  ^~~~~~~
  s6_addr
/home/ssuslov/src/PcapPlusPlus/Dist/header/IpUtils.h:70:2: error: ‘in6_addr’ does not name a type; did you mean ‘s6_addr’?
  in6_addr* sockaddr2in6_addr(struct sockaddr *sa);
  ^~~~~~~~
  s6_addr
/home/ssuslov/src/PcapPlusPlus/Dist/header/IpUtils.h:84:23: error: ‘in_addr’ was not declared in this scope
  uint32_t in_addr2int(in_addr inAddr);
                       ^~~~~~~
/home/ssuslov/src/PcapPlusPlus/Dist/header/IpUtils.h:84:23: note: suggested alternative: ‘s6_addr’
  uint32_t in_addr2int(in_addr inAddr);
                       ^~~~~~~
                       s6_addr
make[1]: *** [Makefile:16: Obj/main.o] Error 1
```

Reply:
I could not fix this problem in different way

Reply:
that is weird because you haven't made any other changes in this file...

Reply:
There is the following code in  `IpUtils.h`:
```c++
#ifdef LINUX
#include <in.h>
#include <arpa/inet.h>
#endif
#ifdef MAC_OS_X
#include <netinet/in.h>
#include <arpa/inet.h>
#endif
#if defined(WIN32) || defined(WINx64) || defined(PCAPPP_MINGW_ENV)
#include <winsock2.h>
#include <ws2tcpip.h>
#endif
#ifdef FREEBSD
#include <sys/socket.h>
#include <netinet/in.h>
#include <arpa/inet.h>
#endif
```

And I think the pre-processor constant `LINUX` is not defined in example for some reason and necessary header files are not included.

Reply:
I think I managed to fix that!

You can remove `IpUtils.h` from `Splitters.h` and add it to `IPPortSplitters.h` below the `#if` clause, like that:

```cpp
#include "Splitters.h"
#if !defined(WIN32) && !defined(WINx64) //for using ntohl, ntohs, etc.
#include <in.h>
#endif
#include "IpUtils.h"
```

I haven't checked that on Windows or Mac though, just on Linux and it's building fine

Reply:
The changes have been made

Reply:
Great thanks!